catalog/schematron/catalog-schematron-plugin/src/main/java/ddf/services/schematron/SchematronValidationService.java

```diff
@@ -85,16 +94,17 @@
  * @author rodgersh
  * @see <a href="http://www.schematron.com">Schematron</a>
  */
-public class SchematronValidationService implements MetacardValidator, Describable {
+public class SchematronValidationService
+        implements MetacardValidator, Describable, ReportingMetacardValidator {
```

Comment:
Should we upgrade to the non-deprecated `Describable` while we're here? https://github.com/codice/ddf/blob/0a87e4db67061f90cc9ed49ad1648ed1ff29b755/platform/platform-api/src/main/java/org/codice/ddf/platform/services/common/Describable.java#L22-L22